analytics_queries/participants.sql

```diff
@@ -36,8 +36,17 @@
             JOIN participant_identities pi on pp.participant_identity_id = pi.id
             LEFT OUTER JOIN ecf_participant_validation_data epvd on pp.id = epvd.participant_profile_id
             LEFT OUTER JOIN ecf_participant_eligibilities epe on pp.id = epe.participant_profile_id
+            JOIN (
+                 SELECT
+                    DISTINCT ON (participant_profile_id) participant_profile_id,
+                    training_status
+                 FROM induction_records
+                 JOIN induction_programmes ON induction_programmes.id = induction_records.induction_programme_id
```

Comment:
Just want to confirm we just want the latest induction record, not scoped to a provider?

Reply:
Hi @cwrw I thought that.. but we don't have the results or joins broken by lead provider anywhere in this query.. so I reckon we want the latest induction record regardless the lead provider..

Reply:
chatted in person but for posterity: We agreed that getting the latest induction will not always yield the latest provider/training status in some edge cases, however if we would like to scope the latest induction record to a provider there might be some changes required on the BQ side. We will need discuss and coordinate that at a later time with the BQ team